azure-pipelines-code-mirror.yml

```
@@ -1,114 +1,123 @@
-      - ${{ if ne(variables['System.TeamProject'], 'DevDiv') }}:
-        - script: |
-            git remote add azdo-mirror https://dn-bot:$(dn-bot-dnceng-build-rw-code-rw)@dev.azure.com/dnceng/internal/_git/$(AzDORepoName)
-          displayName: Add Azure DevOps remote (DncEng)
-          workingDirectory: $(WorkingDirectoryName)
-      - script: |
-          git reset --hard origin/$(BranchToMirror)
-        displayName: Hard reset local branch to GitHub branch
-        workingDirectory: $(WorkingDirectoryName)
-      - powershell: |
-          git push azdo-mirror $(BranchToMirror) --tags $(ExtraPushArgs)
-
-          if ($LASTEXITCODE -EQ 0) {
-            Write-Host "Push was successful"
-            exit
-          }
-
-          git fetch azdo-mirror
-          git fetch origin
-          $commits = (git --no-pager rev-list origin/$(BranchToMirror)..azdo-mirror/$(BranchToMirror) | Measure-Object -line).Lines
-          if ($commits -NE 0) {
-            Write-Host "##vso[task.LogIssue type=error;]Mirror repository $(AzDORepoName) has unexpected commits"
-            git --no-pager log origin/$(BranchToMirror)..azdo-mirror/$(BranchToMirror)
-            exit 1
-          }
-
-          Write-Host "##vso[task.LogIssue type=warning;]Push failed for unknown reason"
-
-          $retryattempt=0
-          while ($retryattempt -LT 3) {
-            $retryattempt+=1
-            Write-Host "Retry attempt $retryattempt of 3 in 5 seconds..."
-            Start-Sleep -Seconds 5
-
-            git push azdo-mirror $(BranchToMirror) $(ExtraPushArgs)
-            if ($LASTEXITCODE -EQ 0) {
-              Write-Host "Push successful"
-              exit
-            }
-          }
-
-          Write-Host "##vso[task.LogIssue type=error;]git failed to push to Azure DevOps mirror"
-          exit 1
-        displayName: Push changes to Azure DevOps repo
-        workingDirectory: $(WorkingDirectoryName)
-
-      - powershell: |
-          $commit = (git rev-parse HEAD).Substring(0, 7)
-          $target = "$(GithubRepo) $(BranchToMirror)".Replace('/', ' ')
+    pool:
+      name: $(DncEngInternalBuildPool)
+      image: windows.vs2019.amd64
+      os: windows
+# Moves code from GitHub into internal repos
+    stages:
+    - stage: Merge_GitHub_to_Azure_DevOps
+      jobs:
+      - template: /eng/common/templates/jobs/jobs.yml@self
+        parameters:
+          enableTelemetry: true
+          helixRepo: dotnet/arcade
+          jobs:
+          - job: Merge_GitHub_to_Azure_DevOps
+            enableSBOM: false
+            pool:
+              ${{ if eq(variables['System.TeamProject'], 'DevDiv') }}:
+                name: AzurePipelines-EO
+                image: 1ESPT-Ubuntu22.04
+                os: linux
+              # If it's not devdiv, it's dnceng
+              ${{ if ne(variables['System.TeamProject'], 'DevDiv') }}:
+                name: $(DncEngInternalBuildPool)
+                image: 1es-ubuntu-2204-pt
```

Didn't see this at first. Let's not use the pt images in new changes, the prereqs are now in the regular 1es images. Also, maybe try mariner so that we future proof this job? 

```suggestion
                image: 1es-mariner-2
```

---

Sure